lib/puppet_litmus/rake_tasks.rb

```diff
@@ -377,6 +377,9 @@ def run_local_command(command)
         # The host_name might be localhost or an IP or a DNS-resolvable node.
         prefix = 'litmus:acceptance:'
         tasks = Rake::Task.tasks.select { |task| task.name =~ %r{^#{prefix}.+:\d+$} }
+        if tasks.empty?
+          tasks = Rake::Task.tasks.select { |task| task.name =~ %r{delivery.puppetlabs.net$$}; }
```

Comment:
Wouldn't it make more sense to extend the regex so it matches also for the vmpooler hostnames?

Reply:
why is it not everything but localhost ?

Reply:
something like `^#{prefix}.\S+` ?

Reply:
no tests :(

Reply:
@tphoney oops adding the tests now

Reply:
@florindragos sure let me try that. Thank you.

Reply:
actually no, this would include `serial` and `parallel`